db/schema.rb

```diff
@@ -577,14 +577,12 @@
     t.bigint "account_id", null: false
     t.string "name", null: false
     t.integer "visibility", default: 0
-    t.bigint "created_by_id", null: false
-    t.bigint "updated_by_id", null: false
+    t.bigint "created_by_id"
+    t.bigint "updated_by_id"
     t.jsonb "actions", default: {}, null: false
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.index ["account_id"], name: "index_macros_on_account_id"
-    t.index ["created_by_id"], name: "index_macros_on_created_by_id"
```

Comment:
Should we keep this?

Reply:
Yes, we are removing the index as it can be a null value.